src/cryptography/hazmat/primitives/ciphers/algorithms.py

```diff
@@ -28,7 +28,8 @@ class AES(CipherAlgorithm, BlockCipherAlgorithm):
     name = "AES"
     block_size = 128
     # 512 added to support AES-256-XTS, which uses 512-bit keys
-    key_sizes = frozenset([128, 192, 256, 512])
+    # 384 added to support AES-192-SIV (SIV uses split keys)
```

Comment:
Why is this needed

Reply:
...It isn't and is a legacy of a separate approach.